docs/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.0.x-to4.0.6.md

```diff
@@ -80,7 +69,7 @@ module.exports = ({ env }) => ({
 ```
 :::
 
-:::
+
```

Comment:
Line 64: maybe we can remove the default value then?
```
keys: env.array('APP_KEYS'),
```

And then precising to put the secrets in the environment variables, typically in the .env file in dev